### PR TITLE
remove preQualification extension, update framework reference

### LINKED ISSUE #217: Remove preQualification extension and update guidance accordingly
Related: https://github.com/open-contracting/standard/issues/906

https://standard.open-contracting.org/profiles/ppp/latest/en/framework/#ii-1-pre-qualification

### PR DESCRIPTION
**Pull request by duncandewhurst**
Closes #217

TO-DO:

- [x] Finalise II.2 (list of pre-qualified suppliers) guidance
- [x] Update [spreadsheet template](https://docs.google.com/spreadsheets/d/1sE3t1d1ZZI3El-VK8GxQgFhh3kueM9OwQxVYw5EMu7k/edit#gid=159397949)
- [x] Update examples
- [x] Update disclosure timing tables in section II
- ~~Update examples in the `ocds-show` repository, copy the files to the `ocds-show-ppp` repository, and then run `make update_examples`~~
- [ ] Update spreadsheet template and worked example URLs

### CODE REVIEW

**Comment by jpmckinney** [comment]
There are a lot of conflicts with 1.0-dev (possibly because this PR also makes whitespace changes). You might want to restart the PR.

**Comment by jpmckinney** [comment]
Changes ignoring whitespace: https://github.com/open-contracting-extensions/public-private-partnerships/pull/245/files?w=1

File: docs/framework.md [hunk 1]
```diff
@@ -937,60 +937,30 @@ References to all signatories to the contract (including any financiers that are
...
-**JSON example:**
+Procurement procedures can involve one or more competitive stages:
 
-```eval_rst
-.. jsoninclude:: examples/full.json
-  :jsonpointer: /releases/7/preQualification/period
-  
-```
+* In a single-stage procedure, the procuring entity invites suppliers to bid without submitting any prior information.
+* In a multi-stage procedure, the procuring entity issues an invitation to participate in the process based on which suppliers submit requests to participate. The procuring entity then assesses the submissions and establishes a list of qualified suppliers to invite to bid.
```
Comment (jpmckinney) [inline docs/framework.md]: This paragraph is confusing.

```suggestion
* In a multi-stage procedure, suppliers submit requests to participate in a first stage. The procuring entity assesses these requests, and establishes a list of qualified suppliers to invite to bid. Qualified supplies then submit bids in a second stage. The procuring entity assesses the bids, and awards a contract to the winning bid.
```

File: docs/framework.md [hunk 2]
```diff
@@ -999,16 +969,16 @@ P   U
 
 </div>
 
-The `preQualification/enquiryPeriod` field should be used to provide the period during which enquiries may be made and answered. 
+Use the `tender/period` field to provide the period during which the first stage is open for submissions, `period/endDate` should contain the closing date for submissions.
```
Comment (jpmckinney) [inline docs/framework.md]:
```suggestion
Use the `tender/period` field to provide the period during which the first stage is open for submissions. `period/endDate` should contain the closing date for submissions.
```

File: docs/framework.md [hunk 3]
```diff
@@ -1151,130 +1121,32 @@ Information on the eligibility criteria for participants in the pre-qualificatio
...
+Request for Qualification (RFQ):
 
 > The set of documents issued by the procuring authority that constitute the basis of the qualification and potentially the pre-selection of candidates (the short list). Qualified (or short-listed candidates) will then be invited to submit a proposal (or to enter into a new phase prior to bid submission, such as a dialogue phase or interactive phase). ([Source](https://ppp-certification.com/ppp-certification-guide/glossary))
 
-Links to these documents should be provided using [Document](reference/documents) objects in the [preQualification/documents](../reference/schema/#release-schema.json,,preQualification/documents) array. Each document's `documentType` field should be set to a value from the [document type codelist](../reference/codelists/#documenttype), to identify the type of document being disclosed.
-
-**Example**: See [section I.4](#i-4-project-economic-and-social-benefits) for JSON and flattened examples of the `documents` building block.
-
-### II.2. List of pre-qualified suppliers 
+In a single-stage procedure, the RFQ documents are embedded in the Request for Proposal documents.
```
Comment (jpmckinney) [inline docs/framework.md]: What is the source of this statement? I wasn't aware that an RFQ could be part of an RFP; typically these are separate stages/processes.

Comment (duncandewhurst) [reply docs/framework.md]: See the entry for Request For Proposal in https://ppp-certification.com/ppp-certification-guide/glossary

Comment (jpmckinney) [reply docs/framework.md]: Aha. NIGP describes RFP and RFQ as separate sub-processes: https://www.nigp.org/resource/global-best-practices/Public-Private%20Partnership%20P3%20Facilities%20and%20Infrastructure%20Best%20Practice.pdf?dl=true

What APMG above is saying is that qualification/selection criteria are included in the RFP documents. I think it's clearer to say that, than to say "RFQ documents are embedded in the RFP documents".

Comment (jpmckinney) [reply docs/framework.md]: Or, this statement from APMG for "One-stage tender" is clearer:

> A tender process in which the government issues its Request for Qualifications (RFQ) and Request for Proposal (RFP) together, and bidders submit their qualifications and bid as a single submission.

The original statement from AMPG for "Request for Proposal (RFP)" was:

> In one-stage open tender processes, the contents of a Request for Qualifications (RFQ) are embedded in the same set of documents, for example, the Request for Proposal (RFP) also includes the RFQ regulations.

This is somewhat different than the profile's text, because "documents embedded in documents" doesn't make sense, but "the contents of a Request for Qualifications (RFQ) are embedded", or "the Request for Proposal (RFP) also includes the RFQ regulations" does make sense.

Comment (duncandewhurst) [reply docs/framework.md]: Good point. I updated the RFQ guidance in https://github.com/open-contracting-extensions/public-private-partnerships/pull/245/commits/475291c062d8b7ed4c3179258825bd557fba155e and added guidance for the RFP, which was missing from the revised guidance, in https://github.com/open-contracting-extensions/public-private-partnerships/pull/245/commits/7a95a3c70b5e7b40b5193b2e90cb962cd09842f6.

**Comment by jpmckinney** [comment]
> There are a lot of conflicts with 1.0-dev (possibly because this PR also makes whitespace changes). You might want to restart the PR.

Just checking that you're confident you can resolve all conflicts, before adding more commits?

**Comment by duncandewhurst** [comment]
> > There are a lot of conflicts with 1.0-dev (possibly because this PR also makes whitespace changes). You might want to restart the PR.
> 
> Just checking that you're confident you can resolve all conflicts, before adding more commits?

Thanks for the reminder, resolved in https://github.com/open-contracting-extensions/public-private-partnerships/pull/245/commits/0ba067b8164ce840f013abf71d901381b7767eb8

**Comment by duncandewhurst** [comment]
@jpmckinney this is now ready for review.

Given that I only remembered about the instructions in the readme after updating the examples directly in this repo, I thought it best for you to check you're happy with the updates in this PR, after which I'll go through the steps to update them in the `ocds-show` and `ocds-show-ppp` repositories.

Similarly, I've [updated a copy](https://docs.google.com/spreadsheets/d/1sE3t1d1ZZI3El-VK8GxQgFhh3kueM9OwQxVYw5EMu7k/edit#gid=263302309) of the spreadsheet template but I'll wait to update the URLs in the documentation until you've checked you're happy with the changes in this PR.

File: docs/examples/full.json [hunk 4]
```diff
@@ -81,14 +81,14 @@
         },
         {
           "id": "XX-GOV-99999999",
-          "name": "National Fibre Infrastructure",
+          "name": "National fiber Infrastructure",
```
Comment (jpmckinney) [inline docs/examples/full.json]: Why? Capitalization is incorrect now.

File: docs/framework.md [hunk 5]
```diff
@@ -939,52 +936,22 @@ References to all signatories to the contract (including any financiers that are
...
-<div class="disclosure-timing">
-
-```eval_rst
-==  ==  ==  ==  ==  ==
-PP  P   A   CC  I   R
-==  ==  ==  ==  ==  ==
-P   U                 
-==  ==  ==  ==  ==  ==
+* In a single-stage procedure, the procuring entity invites suppliers to bid without submitting any prior information.
+* In a multi-stage procedure, suppliers submit requests to participate in a first stage. The procuring entity assesses these requests, and establishes a list of qualified suppliers to invite to bid. Qualified supplies then submit bids in a second stage. The procuring entity assesses the bids, and awards a contract to the winning bid.
```
Comment (jpmckinney) [inline docs/framework.md]:
```suggestion
* In a single-stage procedure, the procuring entity invites suppliers to submit bids, without submitting any prior information.
* In a multi-stage procedure, the procuring entity invites suppliers to submit requests to participate in a first stage (pre-qualification). The procuring entity assesses the requests, and establishes a list of qualified suppliers to invite to submit bids. Qualified suppliers then submit bids in a second stage. The procuring entity assesses the bids, and awards a contract to the winning bidder.
```

File: docs/framework.md [hunk 6]
```diff
@@ -939,52 +936,22 @@ References to all signatories to the contract (including any financiers that are
...
-**JSON example:**
-
-```eval_rst
-.. jsoninclude:: examples/full.json
-  :jsonpointer: /releases/7/preQualification/period
-
-```
-
-**Flattened example** (showing top-level fields only)
+In OCDS, the `tender` section represents the invitation to participate and is also used to describe the procedure as a whole.
```
Comment (jpmckinney) [inline docs/framework.md]:
```suggestion
In OCDS, the `tender` section represents the invitation to participate (whether to submit bids, or to submit requests to participate) and is also used to describe the procedure as a whole.
```

File: docs/framework.md [hunk 7]
```diff
@@ -939,52 +936,22 @@ References to all signatories to the contract (including any financiers that are
...
 
-```eval_rst
-.. jsoninclude-flat:: examples/full.json
-  :jsonpointer: /releases/7/preQualification/period
-  :ignore_path: /releases/7/
-```
+> The act of testing prospective bidders to determine whether they meet the pass/fail qualification criteria in advance of issuing the request for proposals. ([Source](https://ppp-certification.com/ppp-certification-guide/glossary))
 
-</div>
+Use the `tender` section to describe the first stage of the contracting process, whether that is the tender itself, or a pre-qualification stage.
```
Comment (jpmckinney) [inline docs/framework.md]:
```suggestion
Use the `tender` section to describe the first stage of the contracting process, whether it involves the submission of bids, or requests to participate.
```

File: docs/framework.md [hunk 8]
```diff
@@ -999,16 +966,15 @@ P   U
 
 </div>
 
-The `preQualification/enquiryPeriod` field should be used to provide the period during which enquiries may be made and answered.
+Use the `tender/tenderPeriod` field to provide the period during which the first stage is open for submissions. `tenderPeriod/endDate` should contain the closing date for submissions.
 
-**Schema**: Information can be provided using a [Period](https://standard.open-contracting.org/1.1/en/schema/reference/#period) object in the [preQualification/enquiryPeriod](../reference/schema/#release-schema.json,,preQualification/enquiryPeriod) field.
+**Schema**: Information can be provided using a [Period](https://standard.open-contracting.org/1.1/en/schema/reference/#period) object in the [tender/period](../reference/schema/#release-schema.json,,tender/period) field.
```
Comment (jpmckinney) [inline docs/framework.md]:
```suggestion
**Schema**: Information can be provided using a [Period](https://standard.open-contracting.org/1.1/en/schema/reference/#period) object in the [tender/tenderPeriod](../reference/schema/#release-schema.json,,tender/tenderPeriod) field.
```

File: docs/framework.md [hunk 9]
```diff
@@ -1151,124 +1116,26 @@ Information on the eligibility criteria for participants in the pre-qualificatio
...
+#### II.1.6. Request for Qualification
 
-**Schema**: Information can be provided using a [Period](https://standard.open-contracting.org/1.1/en/schema/reference/#period) object in the [tender/enquiryPeriod](../reference/schema/#release-schema.json,,tender/enquiryPeriod) field.
-
-**JSON example:**
-
-```eval_rst
-.. jsoninclude:: examples/full.json
-  :jsonpointer: /releases/1/tender/enquiryPeriod
+Request for Qualification (RFQ):
```
Comment (jpmckinney) [inline docs/framework.md]: Do we need this line, which repeats the heading?

File: docs/framework.md [hunk 10]
```diff
@@ -1458,55 +1276,39 @@ P   U
...
-
-```eval_rst
-.. jsonschema:: _static/patched/release-schema.json
-    :include: tender/eligibilityCriteria
-    :collapse:
-    :nocrossref:
-```
-
-**JSON example:**
+Use [Document](reference/documents) objects in the [tender/documents](../reference/schema/#release-schema.json,,tender/documents) array to provide links to procurement documents, feasibility studies, including land acquisition, social, environmental, and rehabilitation related information and reports of independent procurement auditors. A short summary of each document can be provided using its `description` field. Each document's `documentType` field should be set to a value from the [document type codelist](../reference/codelists/#documenttype), to identify the type of document being disclosed.
```
Comment (jpmckinney) [inline docs/framework.md]: The "including land acquisition, social, environmental, and rehabilitation related information" in the middle of a list of document types reads a bit odd.

File: docs/framework.md [hunk 11]
```diff
@@ -1458,55 +1276,39 @@ P   U
...
-.. jsoninclude:: examples/full.json
-  :jsonpointer: /releases/1/tender
-  :include_only: eligibilityCriteria
+**Example**: See [section I.4](#i-4-project-economic-and-social-benefits) for JSON and flattened examples of the `documents` building block.
 
-```
+### II.2. List of pre-qualified suppliers
 
-**Flattened example** (showing top-level fields only)
+Pre-qualification or shortlist.
```
Comment (jpmckinney) [inline docs/framework.md]: Should this be:

```suggestion
> Pre-qualification or shortlist.
```

**Review by jpmckinney — COMMENTED** [review]
Can you summarize the significant changes to the JSON files? They're hard to review.

**Comment by jpmckinney** [comment]
> Given that I only remembered about the instructions in the readme after updating the examples directly in this repo, I thought it best for you to check you're happy with the updates in this PR, after which I'll go through the steps to update them in the `ocds-show` and `ocds-show-ppp` repositories.

That's fine - as long as we keep it all in sync to avoid overwriting changes in future.

File: docs/examples/full.json (hunk 4 above)
Comment (duncandewhurst) [reply docs/examples/full.json]: Whoops. Find and replace error in the course of switching to American spelling. Fixed in https://github.com/open-contracting-extensions/public-private-partnerships/pull/245/commits/5cfeab17c1996768ef7130bee4178189f885455d

File: docs/framework.md (hunk 10 above)
Comment (duncandewhurst) [reply docs/framework.md]: Yes, it's based on this part of the framework which also reads oddly:

> links to all procurement documents, final feasibility study(including cost-benefit analysis, if any), land acquisition, rehabilitation, social (including information on human rights impact assessment) and environmental assessment, related information, and reports of independent procurement auditors (if any).

How about:

```markdown
Use [Document](reference/documents) objects in the [tender/documents](../reference/schema/#release-schema.json,,tender/documents) array to provide links to documents.

You should provide links to:

* all procurement documents 
* feasibility studies, including cost-benefit analyses, if available
* land acquisition, rehabilitation, social and environmental assessments, including information on human rights impact assessments
* reports of independent procurement auditors

A short summary of each document can be provided using its `description` field. Each document's `documentType` field should be set to a value from the [document type codelist](../reference/codelists/#documenttype), to identify the type of document being disclosed.
```

File: docs/framework.md (hunk 9 above)
Comment (duncandewhurst) [reply docs/framework.md]: It's in keeping with the pattern used in other headings: provide the name used in the framework under the OCDS for PPPs heading. Although, in this case, the two are identical. Happy to remove it if you prefer?

File: docs/framework.md (hunk 11 above)
Comment (duncandewhurst) [reply docs/framework.md]: I don't think so. The names from the framework provided under other headings aren't quoted. Only definitions from other sources are quoted. However, perhaps we could make it clearer that the names are taken directly from the framework.

**Comment by duncandewhurst** [comment]
> Can you summarize the significant changes to the JSON files? They're hard to review.

I should've mentioned that I used OCDS Kit to generate `full_record_package.json` from `full.json` so only the latter requires a detailed review.

The main change is to adapt the example to reflect the changes to the framework reference.

Besides that, I fixed some issues:

* Use American spelling of 'Fiber'
* Correct identifiers which were incorrectly converted to numbers
* Correct inconsistent release dates
* Correct inconsistent timezones
* Reduce repetition between releases to make the example easier to maintain
* Use sequential identifiers rather than '123456'
* Add `name` to `unit`

In the course of the updates `parties/identifier/id` and `parties/identifier/scheme` were reordered for some reason.

File: docs/framework.md (hunk 9 above)
Comment (jpmckinney) [reply docs/framework.md]: I suggest either removing it or putting it inside the following blockquote, so that it's clear it is being quoted from the framework.

File: docs/framework.md (hunk 10 above)
Comment (jpmckinney) [reply docs/framework.md]: That's much clearer. I think we can change one bullet to:

```
* land acquisition, rehabilitation, social, human rights, and environmental assessments
```

The original wording I guess treats human rights assessments as a sub-type of social assessment, but I don't think this concept hierarchy is important to retain.

A clearer, easier to read, but longer format would be:

```
* land acquisition assessments
* rehabilitation assessments
* social assessments
* human rights assessments
* environmental assessments

File: docs/framework.md (hunk 11 above)
Comment (jpmckinney) [reply docs/framework.md]: I think it would be clearest for them all to be indicated somehow. I think quoting is fine for that purpose.

**Comment by jpmckinney** [comment]
Thanks! `full.json` is a bit long to review, so I'll trust the changes based on your description.

File: docs/framework.md (hunk 10 above)
Comment (duncandewhurst) [reply docs/framework.md]: Updated in https://github.com/open-contracting-extensions/public-private-partnerships/pull/245/commits/ef0651de03805ab5f86c3493e16fb4452b0422ae

File: docs/framework.md (hunk 9 above)
Comment (duncandewhurst) [reply docs/framework.md]: Removed in https://github.com/open-contracting-extensions/public-private-partnerships/pull/245/commits/ef0651de03805ab5f86c3493e16fb4452b0422ae

File: docs/framework.md (hunk 11 above)
Comment (duncandewhurst) [reply docs/framework.md]: In https://github.com/open-contracting-extensions/public-private-partnerships/pull/245/commits/ef0651de03805ab5f86c3493e16fb4452b0422ae I've quoted all definitions from the framework and removed any which simply repeat the headings.

Comment (jpmckinney) [reply docs/framework.md]: I think maybe this specific one isn't quoted yet.

**Review by jpmckinney — APPROVED** [review]

**Comment by jpmckinney** [comment]
> Update spreadsheet template and worked example URLs

Does this need to be done in this PR, or separately?

> Update examples in the ocds-show repository, copy the files to the ocds-show-ppp repository, and then run make update_examples

I'm going to have another look at this. Maybe we can just delete them from those repositories.

**Comment by jpmckinney** [comment]
> > Update examples in the ocds-show repository, copy the files to the ocds-show-ppp repository, and then run make update_examples
> 
> I'm going to have another look at this. Maybe we can just delete them from those repositories.

I've just deleted the examples from those other repositories, where they were unused.

The instruction about the examples was a hold-over from when ocds-show-ppp was entirely copied into this repository, which caused some real maintenance issues (changes here not being replicated to ocds-show or ocds-show-ppp). The examples themselves serve no purpose in those other repositories, so it's fine to edit them here only.

For reference: https://github.com/open-contracting-extensions/public-private-partnerships/pull/220

**Review by jpmckinney — APPROVED** [review]
